### DISCUSSION TITLE
Storj bucket hyperlinking

### DISCUSSION THREAD

**Discussion by p145085** [community/community#32207] [Other Feature Feedback, Questions, & Ideas] [answered]
I uploaded a '.gif' file to a Storj bucket and hyperlinked it in my 'README.md' but it doesn't appear to load. Was wondering whether links there were blacklisted or what else was up. 
The input I provide looks like follows: 
`![](https://link.storjshare.io/jwnwn23qvtwrsemvqmnqc5ydek5a/gif-for-github%2FZwqV1LU.gif)`

**Comment by radar-blip** [comment, marked as answer]
You need to insert some text between the brackets ```[Image name]```, and remove the ```!```
That will get you a link to your image. >  [Image](https://link.storjshare.io/jwnwn23qvtwrsemvqmnqc5ydek5a/gif-for-github%2FZwqV1LU.gif)
It would be easier to just save the image, and drag (or upload) it into your README.md

**Reply by p145085** [reply]
Ah, you're right, it's a lot easier to upload directly to the README.md. I just didn't know that was an option, thanks for that.

**Reply by radar-blip** [reply]
No problem, glad I could help.

**Comment by lucastorrescarneiro** [comment]
sou um usuario novo e nao consigo nem na página de programação... como faço?

**Reply by radar-blip** [reply]
Programming page? What do you mean?
